src/options/quantifiers_options.toml

```diff
@@ -1995,3 +1995,39 @@ header = "options/quantifiers_options.h"
   type       = "bool"
   default    = "false"
   help       = "Enable SyGuS instantiation quantifiers module"
+
+[[option]]
+  name       = "sygusInstScope"
+  category   = "regular"
+  long       = "sygus-inst-scope=MODE"
+  type       = "SygusInstScope"
+  default    = "INSIDE"
+  help       = "select scope of ground terms"
+  help_mode  = "scope for collecting ground terms for the grammar."
+[[option.mode.INSIDE]]
+  name = "inside"
+  help = "use ground terms inside quantifier bodies only."
```

Comment:
I assume this means that grammar uses ground terms inside the *given* quantifier? Clarify this here.

Reply:
done